### ISSUE TITLE
collapse bar charts based on metadata categories

### ISSUE THREAD

**Issue by jennomics** [PitchInteractiveInc/Phinch#44]
I would like to be able to collapse or merge bar charts based on my metadata categories. For example, if I have a male/female category, I'd like to be able to produce 2 bar charts that summarize the data for those categories. Is this currently a feature?

**Comment by hollybik** [comment]
Right now you would need to use the donut charts to summarize based on non-numerical metadata such as male/female labels (but that doesn't give you a bar chart - I agree that we should add this feature to the taxonomy bar charts in the future).

Two options - you could re-code Male/Female as numerical values (e.g. 1 and 0) in your mapping file and then do "biom add-metadata" to regenerate the file for use in Phinch. Then you'd be able to view the summarized bar charts with using these numerical values in the "Attributes Column Chart"

The second option is to summarize by category command in QIIME, so first run:

summarize_otu_by_cat.py -m mapping_file.txt -i originalBIOM.biom -c MappingFileColumnName -o SummarizedBIOM.biom

Then you'll need to do "biom add-metadata" again, with a new mapping file containing only two lines (one for male, one for female since these would be your new sample IDs I think

I know its a workaround, but hope this helps?

**Comment by jennomics** [comment]
Phinch gets hung up when I try to load the merged table. It works for the
QIIME summarize_taxa_through_plots.py script, so I know it's in good
shape...

**Comment by hollybik** [comment]
Can you just clarify what you mean by "merged table"? Do you mean that you tried the second option I mentioned, the summarize_otu_by_cat.py? Not sure what the issue is here...if you're still having problems you can email me the problematic BIOM file and I'll look into this further.

**Comment by jennomics** [comment]
Yep, I tried the second option. I'll send it tomorrow. Well, later..,
